lib/datadog/appsec/contrib/rails/auto_instrument_railtie.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AppSec
+    module Contrib
+      module Rails
+        # Railtie to include AutoInstrumentation in rails loading
+        class AutoInstrumentRailtie < ::Rails::Railtie
+          # we want to load before config initializers so that any user supplied config
+          # in config/initializers/datadog.rb will take precedence
+          initializer 'datadog.start_appsec', before: :load_config_initializers do
```

Comment:
Should we go with a narrowed scope and specific to action name instead? WDYT?

```suggestion
          initializer 'datadog.appsec.auto_instrument', before: :load_config_initializers do
```